rpm/embedlite-components-qt5.spec

```diff
@@ -1,3 +1,20 @@
+# Don't depend on private xulrunner-qt5 libraries.
+%global privlibs             libfreebl3
+%global privlibs %{privlibs}|libmozalloc
+%global privlibs %{privlibs}|libmozsqlite3
+%global privlibs %{privlibs}|libnspr4
+%global privlibs %{privlibs}|libnss3
+%global privlibs %{privlibs}|libnssdbm3
+%global privlibs %{privlibs}|libnssutil3
+%global privlibs %{privlibs}|libplc4
+%global privlibs %{privlibs}|libplds4
+%global privlibs %{privlibs}|libsmime3
+%global privlibs %{privlibs}|libsoftokn3
+%global privlibs %{privlibs}|libssl3
+
+%global __provides_exclude ^(%{privlibs})\\.so
```

Comment:
Ah, right, only requires is needed, I did q quick fixup so the patch that landed does not have the provides section. Thanks.